### PR TITLE
Enable HA for DDD by default

### PR DESCRIPTION
**Pull request by DanRoseus**
But allow DDD to run without HA as an option for the SVT team

Tested on fermi-dan-ddd-ha-5 :
image

### CODE REVIEW

File: DrivewayDentDeletion/Operators/cicd-test/cicd-tasks.yaml [hunk 1]
```diff
@@ -16,33 +16,9 @@ spec:
       description: The PVC to use for layers
   steps:
     - name: pull-retag-push-to-test
-      image: quay.io/buildah/stable:latest
-      securityContext:
-        privileged: true
+      image: quay.io/openshift/origin-cli:4.6.0
```
Comment (Kinshuk1993) [inline DrivewayDentDeletion/Operators/cicd-test/cicd-tasks.yaml:19]: Do we need to fix the cli version to 4.6.0 or can we use the latest?

File: products/bash/release-apic.sh [hunk 2]
```diff
@@ -22,19 +22,21 @@
...
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
 namespace="cp4i"
 release_name="ademo"
 tracing="false"
 production="false"
 CURRENT_DIR=$(dirname $0)
 
-while getopts "n:r:tp" opt; do
+while getopts "a:n:r:tp" opt; do
```
Comment (Kinshuk1993) [inline products/bash/release-apic.sh:39]: Is `a:` needed here? I am assuming this is for HA_ENABLED but could not find that variable, may be missing?

Comment (DanRoseus) [reply products/bash/release-apic.sh]: Nope. We originally did the HA stuff here, then I moved it to configure-apic-v10.sh, must have missed reverting this change. Now done!

File: products/bash/release-mq.sh [hunk 3]
```diff
@@ -358,3 +381,4 @@ EOF
     divider
   done
 fi
+``
```
[the remark this answers is not in the document]
Comment (DanRoseus) [reply products/bash/release-mq.sh]: Nope, removed!

File: DrivewayDentDeletion/Operators/cicd-test/cicd-tasks.yaml (hunk 1 above)
Comment (DanRoseus) [reply DrivewayDentDeletion/Operators/cicd-test/cicd-tasks.yaml:19]: We're still using 4.6.0 everywhere (not just here). I don't think we can use latest, but probably could move this up a version. I think that's not for this PR though.

**Review by Kinshuk1993 — APPROVED** [review]
looks good

**Pull request merged**